draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -2058,7 +2062,7 @@ site being browsed. Some block cookies based upon user agent cookie policy and/o
 While this document does not endorse or require a specific approach, it is RECOMMENDED that user
 agents adopt a policy for third-party cookies that is as restrictive as compatibility constraints
 permit. Consequently, resources cannot rely upon third-party cookies being treated consistently by
-user agents for the foreseeable future.
+user agents for the foreseeable future. TODO
```

Comment:
> 4. Section 7.1, last para states that "it is RECOMMENDED that user agents adopt
a policy for third-party cookies that is as restrictive as compatibility
constraints permit". This gives no concrete recommendations, and no examples,
thus making this requirement vague and subjective. I think this requirement
should be provided with more details how to deal with it.

My draft response for this is:

>This is difficult to do given the range of compatibility constraints among user agents. Some agents could strive for maximum compatibility while others for maximum user privacy. The third-party cookie policies will differ greatly between those two types of UAs.

Curious if anyone has other thoughts on how to handle this.